src/components/ActNowButtonCard/index.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import { Card, CardContent, Typography, Grid, Button } from "@material-ui/core";
+import UpdateFormDialog from "@/promisetracker/components/FormDialog/UpdateDialog";
+
+import AllInclusiveIcon from "@material-ui/icons/AllInclusive";
+import ChatBubbleOutlineIcon from "@material-ui/icons/ChatBubbleOutline";
+import ControlPointIcon from "@material-ui/icons/ControlPoint";
+import NotificationsNoneIcon from "@material-ui/icons/NotificationsNone";
+import ShareIcon from "@material-ui/icons/Share";
+
+import ConnectCard from "./ConnectCard";
+import ShareCard from "./ShareCard";
+import FollowCard from "./FollowCard";
+import PetitionCard from "./PetitionCard";
+
+import useStyles from "./useStyles";
+
+const ActNowButtonCard = ({ ...props }) => {
+  const [open, setOpen] = useState(false);
+
+  const selectedCard = {
+    CONNECT: "connect",
+    FOLLOW: "follow",
+    PETITION: "petition",
+    SHARE: "share",
+    ACT: "act",
+  };
+
+  const [card, setSelectedCard] = useState(selectedCard.ACT);
```

Comment:
```suggestion
  const [selectedCard, setSelectedCard] = useState(selectedCard.ACT);
```
Isn't ☝🏾  the `useState` naming pattern i.e. `xxx, setXxx`?